src/bundlers/dart/dartMain.ts

```diff
@@ -42,8 +41,7 @@ void main() async {
               }
 
               final result = await service.{{name}}();
-              final json = jsonEncode(result);
-              response = '{"jsonrpc": "2.0", "result": $json, "id": 0}';
+              response = '{"jsonrpc": "2.0", "result": result, "id": 0}';
```

Comment:
I am not 100% sure about this change. Did you test it?

Reply:
I tested (manually) and with integration tests and it seems fine.

Before returning, `response` is going to be encoded in JSON, so IMO it should be fine. [1]

[1] https://github.com/Genez-io/genezio/pull/301/files#diff-b820cfef641b9ca39114a8bc7046ce73e8d8933e6eebfb97ec3831b4f26a7ae1R186